### ISSUE TITLE
Add me to the NikGapps Elite

### ISSUE THREAD

**Issue by Jaypee1er** [nikgapps/config#11629]
Hi, currently am a patreon donator. 

I would like to apply for the products you offer to upgrade my phone (if that makes sense). I currently have a Chinese version oppo x7 which does not have Android auto 'signed' so is not able to use on my car sound system. Would like to make it global version and whatever else you can recommend.

Is there tutorials on how to install the updates as well? 

Thankyou.

**Comment by nikhilmenghani** [comment]
Hi, @Jaypee1er if you follow https://www.youtube.com/watch?v=jZWR9Wz7hMk, you will find how you can have a personalized build created for you. All you need to do is ensure you set AndroidAuto=1 in your config file and submit a pull request. doing so will get you a zip file which you can flash in recovery after flashing any custom Rom for your device. Feel free to reply here or on telegram (for quicker response) and let me know what you'd like to understand.

**Comment by nikhilmenghani** [comment]
@Jaypee1er hi, let me know if you need any assistance!

**Comment by Jaypee1er** [comment]
Hi mate, sorry about late response, just crazy busy at work. I noticed in the video that you scroll down and change to 0 if you don't need the app and 1 if you do want the app. My question is: if I already have calculator and some stuff on my phone, do I change those to a 0 seeing as they are on my phone already?

Hopefully this makes sense. Thankyou

**Comment by nikhilmenghani** [comment]
> Hi mate, sorry about late response, just crazy busy at work. I noticed in the video that you scroll down and change to 0 if you don't need the app and 1 if you do want the app. My question is: if I already have calculator and some stuff on my phone, do I change those to a 0 seeing as they are on my phone already?
> 
> Hopefully this makes sense. Thankyou

Yes, there are apps that comes with your Rom which we call aosp apps. Such as calculator, calendar, contacts, etc. With NikGapps you get google versions of those apps such as Google Calculator, Google Calendar, Google Contacts and so on.... The difference in functionality is minimal and as long as you're happy with the stock aosp version, you can choose to skip the apps by setting them as 0 in your config.

**Comment by Jaypee1er** [comment]
Hi mate, apologies again for late response. Do you offer packages with any AI Tools? I am currently trying to source one that helps with meetings at work. Speech to text and then being able to edit the text ie simplify the meeting into bullet points. 

And any other useful AI tools...

Thank you

**Comment by Jaypee1er** [comment]
https://www.patreon.com/NikGapps/redeem/f/303F6DDDAC

**Comment by nikhilmenghani** [comment]
@Jaypee1er what is the best way to reach you? I am not able to communicate with you to help you
Does telegram work for you?

**Comment by nikhilmenghani** [comment]
@Jaypee1er - please reopen this issue when you want to discuss further.
